Modules/Core/Common/src/itkExceptionObject.cxx

```diff
@@ -41,11 +41,8 @@ class ExceptionObject::ExceptionData
     , m_File(std::move(file))
     , m_Line(line)
   {
-    std::ostringstream loc;
-
-    loc << ':' << m_Line << ":\n";
     m_What = m_File;
-    m_What += loc.str();
+    m_What += ':' + std::to_string(m_Line) + ":\n";
     m_What += m_Description;
```

Comment:
Note that historically, the `what()` text of an `itk::ExceptionObject` includes the file name and the line number, but _not_ the location, `m_Location` (typically the name of the throwing function). I think it would be nice to add this location to the `what()` text. What do you think? Maybe in another pull request...

Reply:
Great idea!